tyler/kotlin/412_fizz-buzz.kt

```diff
@@ -0,0 +1,48 @@
+class Solution {
+    /**
+     * Given a non-negative [Int], maps a list of [String] from 1 to [n] (inclusive).
+     *
+     * https://leetcode.com/problems/fizz-buzz
+     * @throws IllegalArgumentException if [n] is out of range.
+     */
+    fun fizzBuzz(n: Int): List<String> {
+        require(n >= CONSTRAINT_MINIMUM) { CONSTRAINT_MINIMUM_EXCEEDED_ERROR }
+        require(n <= CONSTRAINT_MAXIMUM) { CONSTRAINT_MAXIMUM_EXCEEDED_ERROR }
+
+        return (1..n).map { dividend ->
```

Comment:
`(1..n)` creates a list of values from 1 to n, inclusive (because I use `..`, exclusive would be `until` - like `1 until n`).

`.map` is a function that iterates over a list and transform / convert / change each value into a different value. In this case, I'm changing `Int` into `String` :)